core-services/prow/02_config/_plugins.yaml

```diff
@@ -2806,6 +2806,9 @@ owners:
     openshift/operator-framework-catalogd:
       owners: DOWNSTREAM_OWNERS
       owners_aliases: DOWNSTREAM_OWNERS_ALIASES
+    openshift/operator-framework-olm:
+      owners: DOWNSTREAM_OWNERS
+      owners_aliases: DOWNSTREAM_OWNERS_ALIASES
```

Comment:
I don't see the `DOWNSTREAM_OWNERS_ALIASES` file in the repo yet, and I also don't see it in your downstream OLM PR. Do you have another PR to add this file somewhere?

Reply:
It's a "just in case we add it" thing.

Reply:
Ah OK. As long as the CI is happy with it I am :)